website/src/main/webapp/website_script.js

```diff
@@ -217,4 +241,31 @@ function getResults() {
 function setUpIndexpage() {
   addIndexListener();
   getClassifications();
+  getLoginStatus();
+}
+
+/**
+ * Fetches the login status of user
+ * if logged in, display logout button
+ * if logged out, display button redirect to login page
+ */
+function getLoginStatus() {
+  return fetch('/login').then(response => response.text()).then(link => {
+    // if user is logged in, server sends the logout link
+    if (link.includes('logout')) {
+      // is logged in 
+      const statusElement = document.getElementById('login-status');
```

Comment:
Optional: Create and send display text along with login from if/else in LoginServlet just to reduce amount of work done on front end side. Definitely not important though.